starwars/src/components/CharacterList.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import Character from "./Character";
+
+export class CharacterList extends Component {
+  render() {
+    return (
+      <div>
+        <ul className="jase">
+           {this.props.cruiser.map(function(publix,index){
+                return <Character key = {index} newAttr={publix}/>
+            })}
+
```

Comment:
Once you get familiar with the data flow, work on variables names. Part of clean coding is descriptive naming so somebody that doesn't know the application can reasonable deduce what is going on